src/com/xilinx/rapidwright/edif/EDIFCellInst.java

```diff
@@ -88,57 +88,66 @@ public EDIFName getViewref() {
     public void setViewref(EDIFName viewref) {
         this.viewref = viewref;
     }
-
+    
     /**
-     * This gets a map of all the port refs on the cell instance.  
-     * @return A map of port ref names to port ref objects.
+     * Creates a new map of all the EDIFPortInst objects stored on this EDIFCellInst.  The new map
+     * contains a copy of EDIFPortInsts available at the time of invocation as returned from 
+     * {@link #getPortInstList()}.      
+     * @return A map of EDIFPortInst names ({@link EDIFPortInst#getName()} to the corresponding objects.
      */
     public Map<String, EDIFPortInst> getPortInstMap(){
```

Comment:
Is this method worth keeping around? Looks like you've eliminated all users of it.

Reply:
We could deprecate it and then, remove in a future release?